### PR TITLE
fix(python): less breaking changes

### PR DESCRIPTION
**Pull request by shortcuts**
## 🧭 What and Why

🎟 JIRA Ticket: -

### Changes included:

A bit of DX improvements by removing some unnecessary breaking changes, which will hopefully help user migrate

### CODE REVIEW

File: tests/output/python/requirements.txt [hunk 1]
```diff
@@ -1,6 +1,6 @@
 argcomplete==3.1.6
 click==8.1.7
 packaging==23.2
-pipx==1.3.3
-platformdirs==4.1.0
+pipx==1.3.2
+platformdirs==4.0.0
```
Comment (millotp) [inline tests/output/python/requirements.txt:5]: it doesn't work with the latest version ?

**Review by millotp — COMMENTED** [review]
nice cleanup !

File: tests/output/python/requirements.txt (hunk 1 above)
Comment (shortcuts) [reply tests/output/python/requirements.txt:5]: hm great catch idk why it pushed this, I believe I ran before main was updated with the deps upgrade?

**Comment by shortcuts** [comment]
will keep it flaky until https://github.com/algolia/api-clients-automation/pull/2441 is merged

**Review by millotp — APPROVED** [review]
gg !

**Pull request merged**